generator/generator.go

```diff
@@ -17,12 +20,20 @@ func (g *Generator) Generate(pkg *Package) error {
 	return g.writeFile(pkg)
 }
 
-func (g *Generator) writeFile(pkg *Package) error {
+func (g *Generator) writeFile(pkg *Package) (err error) {
 	file, err := os.Create(g.filename)
 	if err != nil {
 		return err
 	}
 
-	defer file.Close()
+	defer func() {
+		file.Close()
+		if err != nil {
+			fmt.Println()
+			fmt.Println("kallax: No file generated due to an occurred error:")
```

Comment:
remove first println and add a `\n` to the start of the second. Why the `:` if nothing else is being printed afterwards?